modules/apps/commerce/headless/headless-commerce/headless-commerce-delivery-catalog-impl/src/main/resources/OSGI-INF/liferay/rest/v1_0/document.properties

```diff
@@ -0,0 +1,8 @@
+#
+# This is a generated file.
+#
+
+api.version=v1.0
+entity.class.name=com.liferay.headless.commerce.delivery.catalog.dto.v1_0.Document
+osgi.jaxrs.application.select=(osgi.jaxrs.name=Liferay.Headless.Commerce.Delivery.Catalog)
+osgi.jaxrs.resource=true
```

Comment:
```
api.version=v1.0
batch.engine.entity.class.name=com.liferay.headless.commerce.delivery.catalog.dto.v1_0.Category
batch.engine.task.item.delegate=true
batch.planner.export.enabled=false
batch.planner.import.enabled=false
entity.class.name=com.liferay.headless.commerce.delivery.catalog.dto.v1_0.Category
osgi.jaxrs.application.select=(osgi.jaxrs.name=Liferay.Headless.Commerce.Delivery.Catalog)
osgi.jaxrs.resource=true
```

@andrea-ale-sbarra , I noticed that my buildREST is generated differently from how some of our other x.properties look like.

Is there additional steps I have to take to generate the following properly?

```
batch.engine.task.item.delegate=true
batch.planner.export.enabled=false
batch.planner.import.enabled=false
```